### PR TITLE
feat:added  card documentation and made structure changes

### PR DESCRIPTION
**Pull request by Aazam10**
Hi I have added documentation for card and made structure changes to my file system please review

### CODE REVIEW

**Comment by Aazam10** [comment]
CSB for card documentation: https://lus8f.csb.app/components/Card/card.html

File: components/typography/typography.html [hunk 1]
```diff
@@ -5,8 +5,8 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Typography</title>
-    <link rel="stylesheet" href="./typography.css" />
-    <link rel="stylesheet" href="../utility.css" />
+    <link rel="stylesheet" href="typography.css" />
+    <link rel="stylesheet" href="/utility.css" />
```
Comment (shaik-saad) [inline components/typography/typography.html:9]: If the file is in same directory then "./file.html" or "file.html". Either use single dot and single slash or don't use them both(if the file is in same directory). Don't use only slash. Just a suggestion.

**Review by shaik-saad — APPROVED** [review]
Added a suggestion, other than that you can go ahead with the merge.

File: components/Alert/alert.css [hunk 2]
```diff
@@ -21,21 +21,21 @@
 }
 .outline-primary {
   border: 1px solid var(--primary-color);
-  background-color: white;
+  background-color: var(--white-color);
   color: var(--primary-color);
 }
 .outline-success {
```
Comment (ruchiragoswami) [inline components/Alert/alert.css:33]: For readability, it is a good practice to leave one blank line between two code-blocks

File: components/Card/card.css [hunk 3]
```diff
@@ -1,6 +1,6 @@
 .card {
   width: 20rem;
-
+  border: 1px solid gray;
```
Comment (ruchiragoswami) [inline components/Card/card.css]: suggestion for not hard-coding the color. Making it a variable makes it more reusable and uniform throughout the site.

**Review by ruchiragoswami — APPROVED** [review]
Good work Aazam!

File: components/Card/card.css (hunk 3 above)
Comment (Aazam10) [reply components/Card/card.css]: I have done the changes thanks for the suggeston will be merging the branch now.

**Pull request merged**